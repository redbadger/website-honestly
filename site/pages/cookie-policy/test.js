import React from 'react';
import { shallow } from 'enzyme';

import Cookies from './index';

describe('site/pages/cookie-policy', () => {
  it('renders correctly', () => {
    expect(shallow(<Cookies />)).toMatchSnapshot();
  });
});
