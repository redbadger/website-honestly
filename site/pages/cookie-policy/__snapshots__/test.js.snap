// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/cookie-policy renders correctly 1`] = `
<Policy
  effectiveDate="24 of May 2018"
  policies={
    Array [
      Object {
        "body": [Function],
        "heading": "Why do you have a Cookies Policy?",
        "number": "01",
      },
      Object {
        "body": [Function],
        "heading": "What are cookies and similar technologies?",
        "number": "02",
      },
      Object {
        "body": [Function],
        "heading": "What if I don’t want cookies and similar technologies?",
        "number": "03",
      },
      Object {
        "body": [Function],
        "heading": "What cookies collect my data?",
        "number": "04",
      },
      Object {
        "body": [Function],
        "heading": "What similar technologies collect my data?",
        "number": "05",
      },
      Object {
        "body": [Function],
        "heading": "What happens if you update your Cookies Policy?",
        "number": "06",
      },
      Object {
        "body": [Function],
        "heading": "What do I do if I have any questions?",
        "number": "07",
      },
    ]
  }
  social={
    Object {
      "altText": "Red Badger logo",
      "description": "Let’s make things better. We are digital transformation experts who innovate and deliver.",
      "metaImage": "meta-image.jpg",
      "title": "Red Badger",
      "url": "cookie-policy",
    }
  }
  title="Cookies Policy"
/>
`;
